packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -63,6 +63,7 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
     private readonly DEFAULT_COMPONENT_HEIGHT: number = 10;
     private readonly DEFAULT_COMPONENT_ROWHEIGHT: number = 20;
     private readonly SCROLLBAR_PADDING: number = 12;
+    private readonly Y_AXIS_WIDTH: number = 40;
```

Comment:
It is very hard to predict the exact width of the axis because D3 calculates the values based on the data. Sometimes it uses a decimal, in which case the point makes the width larger.
I run tests and 40 is a good compromise that will show the axis without taking excessive real state.

Reply:
OK, we will see later if it needs to be updated. I guess I am used to older, superior SWT technology ;) where the components automatically take their preferred size based on their content ;)